src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -176,11 +176,28 @@ public static void ProcessLogFile(string filePath, Solution solution, string too
                 log = JsonConvert.DeserializeObject<SarifLog>(logText);
             }
 
-            ProcessSarifLog(log, outputPath, solution);
+            ProcessSarifLog(log, outputPath, solution, showMessageOnNoResults: promptOnLogConversions);
 
             SarifTableDataSource.Instance.BringToFront();
         }
 
+        /// <summary>
+        /// Closes the specified SARIF log in the viewer.
+        /// </summary>
+        /// <param name="paths">The complete path to the SARIF log file.</param>
+        public static void CloseSarifLogs(IEnumerable<string> paths)
+        {
+            SarifTableDataSource.Instance.CleanErrors(paths);
```

Comment:
>CleanErrors [](start = 42, length = 11)

Another long overdue change. Nice.

Reply:
No problem.

---
In reply to: [440859935](https://github.com/microsoft/sarif-visualstudio-extension/pull/143#discussion_r440859935) [](ancestors = 440859935)